srcv2/components/Selector.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+const Selector = ({ title, renderOptions, openOptions, setOpenOptions, loadingOptions }) => {
+    return (
+        <div>
+            <button type='button' className='border border-light rounded-lg px-4 py-1 w-full' onClick={() => setOpenOptions(!openOptions)}>
+                <div className='grid grid-flow-col auto-cols-max flex justify-between'>
+                    <p className='text-light text-left'>{title}</p>
```

Comment:
NIT: Light text is not blending well with the white background
```suggestion
                    <p className='text-black text-left'>{title}</p>
```